tests/defaults.yml

```diff
@@ -12,3 +12,41 @@ parameters:
           irrelevant-user: {}
       empty-group:
         users: {}
+
+    secrets:
+      oidc-client:
+        clientSecret: '?{vaultkv:${cluster:tenant}/${cluster:name}/keycloak-auth/clientSecret}'
+      # this is the secret name in case if `bindPassword.name` is used:
+      #ldap-bind:
+      #  bindPassword: '?{vaultkv:${cluster:tenant}/${cluster:name}/ldap-auth/bindPassword}'
+
+    identityProviders:
+      ldap-auth:
+        name: Company LDAP
+        type: LDAP
+        ldap:
+          url: "ldaps://ldap.company.tld:636/ou=services,dc=company,dc=tld?uid"
+          bindDN: "uid=service,ou=idp,dc=company,dc=tld"
+          ca: |-
+            -----BEGIN CERTIFICATE-----
+            -----END CERTIFICATE-----
+          sync: {}
```

Comment:
Just a thought: Might be worth adding a simple LDAP sync config in the default test to verify that the sync config is compiled correctly as well.